spec/factory_specs/contacts_spec.rb

```diff
@@ -1,10 +0,0 @@
-require "rails_helper"
```

Comment:
Personally, I don't like factory specs either. But its weird that the linter told us to delete tests

Reply:
Oh sorry... I didn't. I just spotted them because they had some lint failures in there and decided to delete them. Probably should have done that in a separate PR or at least mentioned it in the description.